packages/kbn-grid-layout/grid/grid_panel/grid_panel.tsx

```diff
@@ -214,21 +191,13 @@ export const GridPanel = forwardRef<HTMLDivElement, GridPanelProps>(
 
     return (
       <div ref={panelRef} css={initialStyles} className="kbnGridPanel">
-        <div
-          css={css`
-            padding: 0;
-            height: 100%;
-            position: relative;
-          `}
-        >
```

Comment:
Removed this wrapper for each panel - it was unnecessary and flattening the HTML as much as possible increases the efficiency.